src/components/data/listing/Listing.js

```diff
@@ -473,6 +492,7 @@ function Listing(props) {
                     }
                     setSharingDlgOpen={setSharingDlgOpen}
                     onPublicLinksSelected={() => setPublicLinksDlgOpen(true)}
+                    onDownloadSelected={() => setDownload(true)}
```

Comment:
So we could put the code in `useEffect` out side just as function and It may work as well but `window` object is available on the client side only. Putting in `useEffect` guarantees that the code will run only in the client-side. But I confess I have limited understanding of SSR. 🤷🏽‍♂️ 
https://medium.com/frontend-digest/why-is-window-not-defined-in-nextjs-44daf7b4604e
https://stackoverflow.com/questions/55151041/window-is-not-defined-in-next-js-react-app